.github/workflows/conformance.yaml

```diff
@@ -0,0 +1,51 @@
+name: Java Conformance CI
```

Comment:
I haven't actually tested that this workflow works corrrectly, because I couldn't figure out how to do that without committing it...

Reply:
Actually, I can see [it is running against this PR](https://github.com/GoogleCloudPlatform/functions-framework-java/actions/runs/668848487). I will work on fixing the issues...

Reply:
Looks like the workflow is working now: https://github.com/GoogleCloudPlatform/functions-framework-java/runs/2151279859?check_suite_focus=true